pkg/kubelet/util/util.go

```diff
@@ -60,3 +65,50 @@ func GetContainerByIndex(containers []v1.Container, statuses []v1.ContainerStatu
 	}
 	return containers[idx], true
 }
+
+var (
+	tmpfsNoswapOptionSupported        bool
+	tmpfsNoswapOptionAvailabilityOnce sync.Once
+)
+
+const TmpfsNoswapOption = "noswap"
+
+func IsTmpfsNoswapOptionSupported(mounter mount.Interface) bool {
+	isTmpfsNoswapOptionSupportedHelper := func() bool {
+		if sysruntime.GOOS == "windows" {
+			return false
+		}
+
+		mountDir, err := os.MkdirTemp("", "tmpfs-noswap-test")
```

Comment:
If you add the dash at the end, then a random suffix will be automatically appended. Might help in some cases where things have failed and you have an orphaned directory left behind (which is not a problem since `/tmp` tends to be cleaned up on startup or in some cases even periodically).

```suggestion
		mountDir, err := os.MkdirTemp("", "tmpfs-noswap-test-")
```